packages/application-shell/src/components/application-shell/application-shell.tsx

```diff
@@ -366,12 +366,15 @@ export const RestrictedApplication = <
                                   return (
                                     <ApplicationContextProvider<AdditionalEnvironmentProperties>
                                       user={user}
-                                      project={project}
```

Comment:
This is important NOT to set it!

The idea is that the application context does not contain the "normal" project permissions within the navbar component, but instead evaluating the permissions for the menu links is done more explicitly and by using the specific permissions for the menu.

We do it this way to be able to use the same "isAuthorized" logic but with different permissions inputs.